### DISCUSSION TITLE
Ignore invalid symbols

### DISCUSSION THREAD

**Discussion by joshuaspear** [bmoscon/cryptofeed#796] [Q&A]
Hi, 

Is there a simple way to ignore invalid symbols for a given exchange, please?

Thanks

**Comment by bmoscon** [comment]
I don't know what this means can you elaborate?

**Reply by joshuaspear** [reply]
Apologies - currently without explicitly adding custom exception handling, if the feedhandler is running but encounters a symbol which is invalid for a given exchange (i.e. an UnsupportedSymbol exception is thrown) the script stops. I wondered whether there was an easy way to handle these exceptions such that any invalid symbols are just ignored?

**Reply by bmoscon** [reply]
that happens at the very start so you should know if there are bad symbols near instantly. You can also check that you are using valid symbols by checking the contents of the data returns by the `.symbols()` method. Its far better to throw an error when an invalid symbols is used that to just silently ignore it
